packages/entities/entities-plugins/src/components/select/PluginSelectCard.vue

```diff
@@ -64,40 +63,38 @@
         </KDropdownMenu>
       </template>
 
-      <template #body>
-        <div
-          class="plugin-card-body"
-          :class="{ 'custom-plugin': isCustomPlugin }"
-          :data-testid="plugin.name"
-          :title="!plugin.available ? t('plugins.select.unavailable_tooltip') : plugin.name"
-          @click="handleCustomClick"
-        >
-          <h4 class="plugin-card-title">
-            {{ plugin.name }}
-          </h4>
-          <PluginIcon
-            :alt="plugin.name"
-            class="plugin-card-icon"
-            :name="plugin.imageName || plugin.id"
-            :size="55"
-          />
-          <p
-            v-if="plugin.description"
-            class="plugin-card-text"
-          >
-            {{ plugin.description }}
-          </p>
-        </div>
-        <div
-          :class="{
-            'plugin-card-create-footer': isCreateCustomPlugin,
-            'plugin-card-footer': !isCreateCustomPlugin,
-          }"
-          @click="handleCustomClick"
+      <div
+        class="plugin-card-body"
+        :class="{ 'custom-plugin': isCustomPlugin }"
+        :data-testid="plugin.name"
+        :title="!plugin.available ? t('plugins.select.unavailable_tooltip') : plugin.name"
+        @click="handleCustomClick"
+      >
+        <h4 class="plugin-card-title">
+          {{ plugin.name }}
+        </h4>
+        <PluginIcon
+          :alt="plugin.name"
+          class="plugin-card-icon"
+          :name="plugin.imageName || plugin.id"
+          :size="55"
+        />
+        <p
+          v-if="plugin.description"
+          class="plugin-card-text"
         >
-          {{ isCreateCustomPlugin ? t('actions.create_custom') : plugin.exists ? t('actions.enabled') : t('actions.enable') }}
-        </div>
-      </template>
+          {{ plugin.description }}
+        </p>
+      </div>
+      <div
+        :class="{
+          'plugin-card-create-footer': isCreateCustomPlugin,
+          'plugin-card-footer': !isCreateCustomPlugin,
```

Comment:
Maybe

```suggestion
```

… is what we intend to have? Since `border-radius` seems to be missing from the **Custom Plugin** card and we can override attributes in `plugin-card-footer` with `plugin-card-create-footer`:

|Custom Plugin|Others|
|:-:|:-:|
|<img width="647" alt="Screenshot 2023-12-04 at 15 06 37" src="https://github.com/Kong/public-ui-components/assets/5277268/ac337bc1-e869-497f-8740-1b55623bcc88">|<img width="647" alt="Screenshot 2023-12-04 at 15 06 49" src="https://github.com/Kong/public-ui-components/assets/5277268/93aa5a56-5e49-4eef-a8f2-b1473ef70d7c">|

Reply:
Added border-radius